Dragonflight/WarriorProtection.lua

```diff
@@ -1463,19 +1471,19 @@ spec:RegisterAbilities( {
             active_dot.thunder_clap = max( active_dot.thunder_clap, active_enemies )
 
             if talent.thunderlord.enabled and cooldown.demoralizing_shout.remains > 0 then
-                reduceCooldown( "demoralizing_shout", min( 4, active_enemies ) )
+                reduceCooldown( "demoralizing_shout", min( 3, active_enemies ) )
             end
 
             if talent.blood_and_thunder.enabled and debuff.rend.up then
                 active_dot.rend = min( active_enemies, 5 )
             end
 
-            if buff.outburst.up then
+            if buff.violent_outburst.up then
```

Comment:
Is overlap between T28 and talent tested?  If you have Violent Outburst and Outburst, are both consumed, or just one of them?  As written, only one of the buffs is consumed here.  Is that correct?

Reply:
Looks like T28 and Talent are BOTH consumed. Tested in-game as well and can verify that both buffs are consumed (if they are up).

![image](https://user-images.githubusercontent.com/5103286/194471791-941058d4-70f0-4d57-b088-c9dd6523336a.png)

Reply:
Fixed in https://github.com/Wyste/hekili/commit/101bcd4e90bfb6ffe25dd8e3fb43c7209307c34e